dev-haskell/floskell/floskell-0.10.5.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+# Tests fail
```

Comment:
In this case, I think the failure is due to missing files from the release tarball: specifically [this directory](https://github.com/ennocramer/floskell/tree/master/styles) appears to be missing.

You can either add this information to your comment here, or try to fetch the files yourself to make the tests pass (as in my other comments, I don't mind trying to fix this myself once I"ve completed this review)